ReadConfig/ServerStatisticsConfig.cs

```diff
@@ -0,0 +1,11 @@
+﻿public class ServerStatisticsConfig
+{
+    public int samplingIntervalSeconds { get; set; }
+    public string serverIdentifier { get; set; }
+
+    public ServerStatisticsConfig(int samplingIntervalSeconds, string serverIdentifier)
```

Comment:
nit: no need for the constructor here, check out the c# object initializer feature.